sc-cli.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
I can totesmcgoats move it to something like cli/ as described above, what do you think about that? In either case, I don't think I can have the commands (which would be part of the cmd package) and the binary (which should be part of main iirc) in the same directory.